Documentation/coding-guidelines/interop-guidelines.md

```diff
@@ -244,3 +248,43 @@ rather than the underlying integral type.
     should be named with the most discoverable name possible; if that name
     is a concept (e.g. Errors), it can be named using managed naming
     guidelines.
+
+
+## UNIX shims
+
+Often, various UNIX flavors offer the same API from the point-of-view of compatibility
```

Comment:
I think it'd be worth noting explicitly that this isn't just about Linux vs OS X vs FreeBSD vs... etc.  It's also about 32-bit vs 64-bit, processor architecture, etc.